src/pal/src/include/pal/context.h

```diff
@@ -468,6 +472,19 @@ inline static void CONTEXTSetPC(LPCONTEXT pContext, DWORD64 pc)
 #endif
 }
 
+inline static DWORD64 CONTEXTGetFP(LPCONTEXT pContext)
```

Comment:
Yea, I noticed that as well. I guess it's fine as is. If it's problematic we can rename it later.